cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -88,17 +88,17 @@ TEST_F(TestProjector, TestProjectCache) {
   // if configuration is different, should return a new projector.
 
   // build a new path by replacing the first '/' with '//'
-  std::string alt_path(GANDIVA_BYTE_COMPILE_FILE_PATH);
-  auto pos = alt_path.find('/', 0);
-  EXPECT_NE(pos, std::string::npos);
-  alt_path.replace(pos, 1, "//");
-  auto other_configuration =
-      ConfigurationBuilder().set_byte_code_file_path(alt_path).build();
-  std::shared_ptr<Projector> should_be_new_projector2;
-  status = Projector::Make(schema, {sum_expr, sub_expr}, other_configuration,
-                           &should_be_new_projector2);
-  ASSERT_OK(status);
-  EXPECT_NE(projector, should_be_new_projector2);
+  // std::string alt_path(GANDIVA_BYTE_COMPILE_FILE_PATH);
+  // auto pos = alt_path.find('/', 0);
+  // EXPECT_NE(pos, std::string::npos);
+  // alt_path.replace(pos, 1, "//");
+  // auto other_configuration =
+  //     ConfigurationBuilder().set_byte_code_file_path(alt_path).build();
+  // std::shared_ptr<Projector> should_be_new_projector2;
+  // status = Projector::Make(schema, {sum_expr, sub_expr}, other_configuration,
+  //                          &should_be_new_projector2);
+  // ASSERT_OK(status);
+  // EXPECT_NE(projector, should_be_new_projector2);
```

Comment:
We don't hash byte_code_file_path anymore, so I plan to remove it.

Reply:
Is it ok if I give you a patch for the test ?